manager-bundle/src/HttpKernel/ContaoCache.php

```diff
@@ -31,45 +31,40 @@ class ContaoCache extends HttpCache implements CacheInvalidation
     /**
      * @var bool
      */
-    private $wasBypassed;
+    private $isDebug;
 
     public function __construct(ContaoKernel $kernel, string $cacheDir = null)
     {
         parent::__construct($kernel, $cacheDir);
 
+        $this->isDebug = $kernel->isDebug();
+
+        $this->addSubscriber(new StripCookiesSubscriber(array_filter(explode(',', getenv('COOKIE_WHITELIST') ?: ''))));
         $this->addSubscriber(new PurgeListener());
         $this->addSubscriber(new PurgeTagsListener());
         $this->addSubscriber(new CleanupCacheTagsListener());
     }
 
-    public function wasBypassed(): bool
```

Comment:
This would be an API break, or was this not yet released?

Reply:
Was not released yet. It's only on master.